packages/cli/test/commands/migrate/init-command.ember-app.test.ts

```diff
@@ -80,3 +81,60 @@ describe('migrate init for ember app variant', () => {
     });
   });
 });
+
+describe('migrate init for ember 4 app', () => {
+  let project: Project;
+  let basePath: string;
+
+  beforeEach(async () => {
+    project = getEmber4AppProject();
+    await project.write();
+    basePath = project.baseDir;
+  });
+
+  afterEach(() => {
+    project.dispose();
+    basePath = '';
+  });
+
+  test(`default run -- ember 4 app`, async () => {
```

Comment:
these tests also need to inspect every new .ts file and confirm the type inference is valid. this can be done with snapshots.

Reply:
This are the tests for migrate init, not migrate. There's no type inference yet at this stage.